docs/user/guides/generate-bindings.md

```diff
@@ -1,75 +1,75 @@
 # Generate Bindings
 
-## Requirements
+Generate a client binding for any Pulp component for the language of your choice.
 
-- Pulp 3 running on `localhost:24817`
-- Docker or Podman
+!!! warning "Language Support"
 
-## Basic Usage
+    Only python and ruby bindings have some testing, although not extensive.
 
+    For python clients, we recommend looking at [pulp-glue] as an alternative.
+    That library is handcrafted, handles tasks for you and knows how to deal with different versions of Pulp and plugins.
 
-The `generate.sh` script takes three positional arguments:
+    If you need to tweak the generator to make it work for your language, have a look at [how it works].
 
-```
-./generate.sh <module-name> <language> [<version>]  
-```
+## Overview
 
-When the optional version parameter is provided, it is used as the version string.
-When it is not provided, the version reported by Pulp's status API is used.
+- Requirements:
+    - Docker/Podman compose
+- General workflow:
+    1. Clone `pulp-openapi-generator`.
+    1. Get the openapi schema for the desired Pulp components.
+    1. Run the generator cli from the the repository
 
-The final packages are stored in the `{plugin_label}-client` directory.
+## 1) Setup
 
-Examples:
+To get started, first clone the repo into your machine.
+It'll be used as the workspace for the bindings generation.
 
 ```bash
-# Create Python Package in 'pulpcore-client/'.
-# Uses API reported  version
-sudo ./generate.sh pulpcore python
+git clone https://github.com/pulp/pulp-openapi-generator
+cd pulp-openapi-generator
+```
 
-# Create Ruby Gem in 'pulp_rpm-client/'.
-# Uses API reported version
-sudo ./generate.sh pulp_rpm ruby
+## 2) Get the schema
 
-# Create Ruby Gem 'pulp_rpm-client/'.
-# Uses '3.0.0rc1.dev.10' version
-sudo ./generate.sh pulp_rpm ruby 3.0.0rc1.dev.10
-```
+Here are some options to get a openapi schema:
 
-## Special Cases
+1. From a python environment with Pulp packages installed:
 
-### Re-Rooted Systems
+    ```bash
+    pulpcore-manager openapi --bindings \
+        --component "core" \
+        --file "core-api.json"
+    ```
 
-During bindings generation the openapi schema is fetched. Use the `PULP_API_ROOT` environment
-variable to instruct the bindings generator where the root of the API is located. For example, the
-default `export PULP_API_ROOT="/pulp/"` is the default root, which then serves the api.json at
-`/pulp/api/v3/docs/api.json`.
+1. From a running pulp instance:
 
-### Remote Systems
+    ```bash
+    PULP_URL="http://localhost:24817/pulp/api/v3/"
```

Comment:
This strange port is meant to be hidden behind a reverse proxy.
Can we just use `http://pulp.example/pulp/api/v3/` ?
"example" is a special TLD to be used in documentation because it will never be resolved on the internet.